javascript/js-in-the-real-world/forms.md

```diff
@@ -1,16 +1,18 @@
 ### Introduction
-Forms are a crucial part of most websites. Almost every major site has sign-up forms, contact forms, search forms and more!  Luckily HTML5 and JavaScript have some handy built-in methods.
 
-- [This tutorial](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation) covers pretty much everything you need to know about HTML form validation.
+Forms are a crucial part of most websites. Almost every major site has sign-up forms, contact forms, search forms and more!  Luckily HTML5 and JavaScript have some handy built-in methods. You've already learned about validation with HTMl and styling validations with CSS in a previous lesson: [Form Validations in the Intermediate HTML/CSS course](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css/lessons/form-validation).
+
+In this lesson, we'll cover the Constraint Validation API: a way to validate forms on the frontend with JavaScript.
+
+- [This tutorial on Form Validation](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation) covers pretty much everything you need to know about HTML form validation. It starts off with HTML attributes, but keep on reading the doc covers the Constraint Validation API.
+- It'll also prove beneficial to go through the [Constraint Validation docs](https://developer.mozilla.org/en-US/docs/Web/API/Constraint_validation).
```

Comment:
These links should not be in the introduction since they come off more as required reading. They should instead be in an Assignment section.

For the Form Validation link, I would just link to where the [Validating forms using JavaScript](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#validating_forms_using_javascript) section begins.

A link that was removed from the HTML/CSS lesson on form validation I think could fit into an assignment section here, too. It was *meant* to be a Youtube video on JS form validation, but for some reason linked to a React based video. What do you think about adding [Web Dev Simplified's JavaScript Form Validation](https://www.youtube.com/watch?v=In0nB0ABaUk) video as an assignment item?

Reply:
> For the Form Validation link, I would just link to where the Validating forms using JavaScript section begins.

Done

> What do you think about adding Web Dev Simplified's JavaScript Form Validation video as an assignment item?

I just watched the video. It's a nice resource on form validation and deserves to be on the curriculum. Though, it doesn't cover anything _new_, which has me hesitant on adding it as an assignment. I'd rather have it as an additional resource.

I'll leave the final call to you

Reply:
If the video doesn't add anything new, I don't think we *need* it as an assignment or additional resource. I'll say we can exclude it for now.